packages/salesforcedx-utils-vscode/test/unit/commands/commandletExecutor.test.ts

```diff
@@ -193,14 +194,14 @@ describe('LibraryCommandletExecutor', () => {
         });
         assert.fail();
       } catch (e) {
-        expect(showErrStub.called).to.be.true;
-        expect(showErrStub.args[0]).to.eql(['Test Command failed to run']);
+        expect(showFailedExecutionStub.called).to.be.true;
+        expect(showFailedExecutionStub.args[0]).to.eql([commandExecutionName]);
```

Comment:
I haven't ran this code & haven't verified, but want to raise the following and make sure this is indeed intended.  Previously, the error was, "Test Command failed to run", and now, the error is only, "Test Command".  This seems like a change in functionality, so I want to point this out and make sure this is correct.

Reply:
good call out.  The file under test hasn't changed at all so the change is only to the test file.  Previously it was making callouts to the internationalization server to get the actual error messages for stubs to return.  Since I updated the stub I changed it to no longer do that in order to avoid worrying about how the localization service handles messages with args in the unit tests for the commandletExecutor.